accounts-db/src/accounts_index/in_mem_accounts_index.rs

```diff
@@ -1283,10 +1283,6 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> InMemAccountsIndex<T,
                                     // If the entry *was* updated, re-mark it as dirty then
                                     // skip to the next pubkey/entry.
                                     let ref_count = v.ref_count();
-                                    if ref_count != 1 || slot_list.len() != 1 {
```

Comment:
if we write slot_list.len() != 1, disk index creates the data file for the bucket.

Reply:
I pushed a commit that allows single_slot_ref_2 to be stored in index bucket.